scala/cask/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.9.9
```

Comment:
maybe this file could dropped.

the sbt version is driven by dockerfile, actually it is `1.10.2`